lib/content-services/content-node-selector/content-node-selector-panel.component.spec.ts

```diff
@@ -483,6 +483,43 @@ describe('ContentNodeSelectorComponent', () => {
                 expect(component.showingSearchResults).toBeFalsy();
             });
 
+            it('should clear the search field, nodes and chosenNode when deleting the search input',  async(() => {
```

Comment:
I know that is largely (wrongly) used in this test file test timeout but can we try to use  fakeasync, tick ?